### PR TITLE
Capstone Project 1 : Chess Event Site

### PR DESCRIPTION
**Pull request by IndieCoderMM**
# Chess event website
For this capstone project, I am building a website for a chess event.
In this branch, I:
- Built mobile-first design for *home-page* and *about-page*
- Modified for the desktop version
- Added mobile menu
- Loaded contents in *feature* section dynamically from JavaScript

Here you can watch my [presentation about this project](https://www.loom.com/share/f776c2ee38fe487b89b186001d697429).

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,553 @@
...
+
+.program-card {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  gap: 0.25rem;
+  padding: 1rem;
+  background-color: rgba(51, 51, 51, 0.879);
+  z-index: 5;
+}
```
Comment (mukhammadrizooff) [inline style.css]: - [ ] [REQUIRED] Good job so far 👍 Please add `hover` effect for cards like [Behance design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) 🙏 

### Your design ❌ 

image

### [Behance design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) ✔️ 

image

File: main.js [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+    bio: 'Nepo is a Russian grandmaster who challenged GM Magnus Calsen for the 2021 World Championship.',
+    profile: 'img/nepo.jpeg',
+  },
+  {
+    name: 'Fabiano Caruana',
+    title: 'Grandmaster (2766)',
+    bio: 'Fabiano, an Italian-American grandmaster, is a chess prodigy who became a grandmaster at age 14.',
+    profile: 'img/fabiano.jpg',
+  },
+];
```
Comment (mukhammadrizooff) [inline main.js]: - [ ] [REQUIRED]  Good job so far! You have done a great job in this section, I would request you to make the following changes to this section. Please, the elements in this section, i.e. the number of players should be `6` but you have `4`, and the `title` part's the bottom and the `bio` part at the top should be a `small line` as given in [Behance design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) 🙏 At least I think it's easy for a smart student like you to do it 💪 

### Your design ❌ 

image

### [Behance design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) ✔️ 

image

**Review by mukhammadrizooff — CHANGES_REQUESTED** [review]
Hi @IndieCoderMM, 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!



## Highlights
- No linter errors ✔️ 
- Good implementations ✔️ 
- Descriptive summary ✔️ 
- Descriptive Presentation video ✔️ 
- Perfect mobile version design ✔️ 

## Suggested changes
_Check the comments under the review._

## Optional suggestions
- [ ] [OPTIONAL] In my opinion, the `width` and `height` of the **Header** section in the desktop version is good, but I think that the `height` value of the other `sections` is low, because the rest of the sections are reflected flat. It would be great if you could also implement this segestion 😃 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Review by emmiiorji — APPROVED** [review]
Hi @IndieCoderMM, 

# Status: Approved ✅

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

congratulations-2805

## Good Highlights✔️

- Linter checks are passing
- Good commit messages
- You have implemented requested changes


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**